recipes/r-base/meta.yaml

```diff
@@ -50,7 +50,7 @@ requirements:
     - curl                     # [not win]
     - xz 5.2.*                 # [not win]
     - openjdk
-    - {{posix}}pkg-config
+    - {{native}}pkg-config
```

Comment:
Why?
conda search m2-pkg-config
m2-pkg-config                0.29.1                        1  defaults

Using the native (i.e. mingw-w64) one instead of the posix one (i.e. msys2) is unlikely to work. This recipes